app/controllers/WhichTaxRegimeController.scala

```diff
@@ -49,23 +49,25 @@ class WhichTaxRegimeController @Inject() (
       val hasSaEnrolment = EnrolmentDef.Sa.findEnrolmentValues(request.enrolments).isSuccess
 
       (hasEpayeEnrolment, hasVatEnrolment, hasSaEnrolment) match {
-        case (true, false, false)                        => Redirect(routes.LandingController.epayeLandingPage)
-        case (false, true, false)                        => Redirect(routes.LandingController.vatLandingPage)
-        case (false, false, true) if appConfig.saEnabled => Redirect(routes.LandingController.saLandingPage)
+        case (true, false, false)                          => Redirect(routes.LandingController.epayeLandingPage)
+        case (false, true, false)                          => Redirect(routes.LandingController.vatLandingPage)
+        case (false, false, true) if appConfig.saEnabled   => Redirect(routes.LandingController.saLandingPage)
+        case (false, false, false) if appConfig.siaEnabled => Redirect(routes.LandingController.siaLandingPage)
```

Comment:
we're gonna fix this up in a later ticket so we don't need to worry about the which tax regime stuff for this ticket